orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/util/PackageInfo.groovy

```diff
@@ -90,62 +90,72 @@ class PackageInfo {
       return request
     }
 
-    String prefix = "${request.package}${versionDelimiter}"
-    String fileExtension = ".${packageType}"
+    def requestPackages = request.package.split(" ")
 
-    Map triggerArtifact = filterArtifacts(triggerArtifacts, prefix, fileExtension)
-    Map buildArtifact = filterArtifacts(buildArtifacts, prefix, fileExtension)
+    requestPackages.eachWithIndex { requestPackage, index ->
 
-    // only one unique package per pipeline is allowed
-    if (triggerArtifact && buildArtifact && triggerArtifact.fileName != buildArtifact.fileName) {
-      throw new IllegalStateException("Found build artifact in Jenkins stage and Pipeline Trigger")
-    }
+      String prefix = "${requestPackage}${versionDelimiter}"
+      String fileExtension = ".${packageType}"
 
-    String packageName
-    String packageVersion
+      Map triggerArtifact = filterArtifacts(triggerArtifacts, prefix, fileExtension)
+      Map buildArtifact = filterArtifacts(buildArtifacts, prefix, fileExtension)
 
-    if (triggerArtifact) {
-      packageName = extractPackageName(triggerArtifact, fileExtension)
-      if (extractVersion) {
-        packageVersion = extractPackageVersion(triggerArtifact, prefix, fileExtension)
+      // only one unique package per pipeline is allowed
+      if (triggerArtifact && buildArtifact && triggerArtifact.fileName != buildArtifact.fileName) {
+        throw new IllegalStateException("Found build artifact in Jenkins stage and Pipeline Trigger")
       }
-    }
 
-    if (buildArtifact) {
-      packageName = extractPackageName(buildArtifact, fileExtension)
-      if (extractVersion) {
-        packageVersion = extractPackageVersion(buildArtifact, prefix, fileExtension)
+      String packageName
+      String packageVersion
+
+      if (triggerArtifact) {
+        packageName = extractPackageName(triggerArtifact, fileExtension)
+        if (extractVersion) {
+          packageVersion = extractPackageVersion(triggerArtifact, prefix, fileExtension)
+        }
       }
-    }
 
-    if (packageVersion) {
-      request.put('packageVersion', packageVersion)
-    }
+      if (buildArtifact) {
+        packageName = extractPackageName(buildArtifact, fileExtension)
+        if (extractVersion) {
+          packageVersion = extractPackageVersion(buildArtifact, prefix, fileExtension)
+        }
+      }
 
-    if (packageName) {
-      request.put('package', packageName)
+      if (packageVersion) {
+        request.put('packageVersion', packageVersion)
+      }
+
+      // If the package cannot be found it's passed down as is
```

Comment:
I think it might be helpful to add some more detail to the comment. Maybe something closer to the commit comment itself?